### PR TITLE
Update docs around the users table to account for Windows

### PR DESCRIPTION
**Pull request by scottlundgren**
#4421 looking for initial review from @zacbrown 

Particularly looking for input on the description.

### CODE REVIEW

**Comment by osqueryer** [comment]
:-1: The commit 31911424718025a0d377417b193a428d9460fcdf (Job results: [3199](https://jenkins.osquery.io/job/osqueryPullRequestOSX/3199/)) failed one or more tests (**macOS/OS X**).

**Comment by osqueryer** [comment]
:no_good: The commit 31911424718025a0d377417b193a428d9460fcdf (Job results: [3666](https://jenkins.osquery.io/job/osqueryPullRequestAudit/3666/)) failed the code audit or documentation test.

**Comment by osqueryer** [comment]
:-1: The commit 31911424718025a0d377417b193a428d9460fcdf (Job results: [4122](https://jenkins.osquery.io/job/osqueryPullRequestWindows/4122/)) failed one or more tests (**Windows**).

**Comment by osqueryer** [comment]
:-1: The commit 31911424718025a0d377417b193a428d9460fcdf (Job results: [7221](https://jenkins.osquery.io/job/osqueryPullRequestLinux/7221/)) failed one or more tests (**Linux**).

**Comment by osqueryer** [comment]
:-1: The commit 31911424718025a0d377417b193a428d9460fcdf (Job results: [4123](https://jenkins.osquery.io/job/osqueryPullRequestWindows/4123/)) failed one or more tests (**Windows**).

**Comment by osqueryer** [comment]
:-1: The commit 31911424718025a0d377417b193a428d9460fcdf (Job results: [1638](https://jenkins.osquery.io/job/osqueryPullRequestFreeBSD/1638/)) failed one or more tests (**FreeBSD**).

**Comment by osqueryer** [comment]
:-1: The commit 31911424718025a0d377417b193a428d9460fcdf (Job results: [1639](https://jenkins.osquery.io/job/osqueryPullRequestFreeBSD/1639/)) failed one or more tests (**FreeBSD**).

**Comment by osqueryer** [comment]
:-1: The commit 31911424718025a0d377417b193a428d9460fcdf (Job results: [7222](https://jenkins.osquery.io/job/osqueryPullRequestLinux/7222/)) failed one or more tests (**Linux**).

**Comment by fmanco** [comment]
ok to test

**Comment by osqueryer** [comment]
:-1: The commit 31911424718025a0d377417b193a428d9460fcdf (Job results: [3200](https://jenkins.osquery.io/job/osqueryPullRequestOSX/3200/)) failed one or more tests (**macOS/OS X**).

**Comment by osqueryer** [comment]
:-1: The commit 31911424718025a0d377417b193a428d9460fcdf (Job results: [3201](https://jenkins.osquery.io/job/osqueryPullRequestOSX/3201/)) failed one or more tests (**macOS/OS X**).

**Comment by osqueryer** [comment]
:no_good: The commit 31911424718025a0d377417b193a428d9460fcdf (Job results: [3667](https://jenkins.osquery.io/job/osqueryPullRequestAudit/3667/)) failed the code audit or documentation test.

**Comment by osqueryer** [comment]
:-1: The commit 31911424718025a0d377417b193a428d9460fcdf (Job results: [4124](https://jenkins.osquery.io/job/osqueryPullRequestWindows/4124/)) failed one or more tests (**Windows**).

**Comment by osqueryer** [comment]
:-1: The commit 31911424718025a0d377417b193a428d9460fcdf (Job results: [7223](https://jenkins.osquery.io/job/osqueryPullRequestLinux/7223/)) failed one or more tests (**Linux**).

**Comment by osqueryer** [comment]
:-1: The commit 31911424718025a0d377417b193a428d9460fcdf (Job results: [4125](https://jenkins.osquery.io/job/osqueryPullRequestWindows/4125/)) failed one or more tests (**Windows**).

**Comment by osqueryer** [comment]
:-1: The commit 31911424718025a0d377417b193a428d9460fcdf (Job results: [1640](https://jenkins.osquery.io/job/osqueryPullRequestFreeBSD/1640/)) failed one or more tests (**FreeBSD**).

**Comment by osqueryer** [comment]
:-1: The commit 31911424718025a0d377417b193a428d9460fcdf (Job results: [1641](https://jenkins.osquery.io/job/osqueryPullRequestFreeBSD/1641/)) failed one or more tests (**FreeBSD**).

**Comment by osqueryer** [comment]
:-1: The commit 31911424718025a0d377417b193a428d9460fcdf (Job results: [7224](https://jenkins.osquery.io/job/osqueryPullRequestLinux/7224/)) failed one or more tests (**Linux**).

**Comment by fmanco** [comment]
😢 tests failing, given the simple commit this is probably jenkins. Need to investigate.

**Comment by fmanco** [comment]
Okay lets try again because tests are passing for other PRs and I still don't get it why they're not passing for this. ok to test

File: specs/users.table [hunk 1]
```diff
@@ -1,5 +1,5 @@
 table_name("users")
-description("Local system users.")
+description("Local user accounts (including domain accounts that have logged on locally (Windows))")
```
Comment (fmanco) [inline specs/users.table]: nit: Finish with a . because it that's the standard for all tables.

**Comment by osqueryer** [comment]
:-1: The commit 31911424718025a0d377417b193a428d9460fcdf (Job results: [7228](https://jenkins.osquery.io/job/osqueryPullRequestLinux/7228/)) failed one or more tests (**Linux**).

**Comment by osqueryer** [comment]
:-1: The commit 31911424718025a0d377417b193a428d9460fcdf (Job results: [1645](https://jenkins.osquery.io/job/osqueryPullRequestFreeBSD/1645/)) failed one or more tests (**FreeBSD**).

**Comment by osqueryer** [comment]
:-1: The commit 31911424718025a0d377417b193a428d9460fcdf (Job results: [1646](https://jenkins.osquery.io/job/osqueryPullRequestFreeBSD/1646/)) failed one or more tests (**FreeBSD**).

**Comment by osqueryer** [comment]
:no_good: The commit 31911424718025a0d377417b193a428d9460fcdf (Job results: [3671](https://jenkins.osquery.io/job/osqueryPullRequestAudit/3671/)) failed the code audit or documentation test.

**Comment by osqueryer** [comment]
:-1: The commit 31911424718025a0d377417b193a428d9460fcdf (Job results: [7229](https://jenkins.osquery.io/job/osqueryPullRequestLinux/7229/)) failed one or more tests (**Linux**).

**Comment by osqueryer** [comment]
:-1: The commit 31911424718025a0d377417b193a428d9460fcdf (Job results: [4129](https://jenkins.osquery.io/job/osqueryPullRequestWindows/4129/)) failed one or more tests (**Windows**).

**Comment by osqueryer** [comment]
:-1: The commit 31911424718025a0d377417b193a428d9460fcdf (Job results: [4132](https://jenkins.osquery.io/job/osqueryPullRequestWindows/4132/)) failed one or more tests (**Windows**).

**Comment by fmanco** [comment]
Let's give it another try to check whether this was just transient. ok to test

**Comment by fmanco** [comment]
Okay, looks that my comment about the missing period is not just a nit, it is mandatory.

**Comment by fmanco** [comment]
@scottlundgren I updated your PR, lets see if this fixes it.

**Comment by fmanco** [comment]
ok to test

**Comment by osqueryer** [comment]
:-1: The commit 31911424718025a0d377417b193a428d9460fcdf (Job results: [7258](https://jenkins.osquery.io/job/osqueryPullRequestLinux/7258/)) failed one or more tests (**Linux**).

**Comment by fmanco** [comment]
Last build of this pr succeeded but was somehow not reported. Why Jenkins why? Let's try again ok to test

**Comment by muffins** [comment]
ok to test

**Comment by osqueryer** [comment]
:-1: The commit 31911424718025a0d377417b193a428d9460fcdf (Job results: [7288](https://jenkins.osquery.io/job/osqueryPullRequestLinux/7288/)) failed one or more tests (**Linux**).

**Comment by fmanco** [comment]
Not sure what's going on here tbh. Jenkins seems to be reporting status logs for the PR but somehow it doesn't get updated. I'll try to close and re-open, otherwise @scottlundgren I think the easiest is if you could just re-submit this.

**Comment by fmanco** [comment]
ok to test

**Comment by fmanco** [comment]
I'm not sure how to fix this. Also tried to force push a squashed commit and the status are still not reported and stuck on `Expected`. So @scottlundgren if you don't mind, please close this PR and open a new one. Thank you!

**Comment by muffins** [comment]
ok to test

**Pull request merged**